web/script.js

```diff
@@ -49,7 +49,7 @@
   }
 
   function makeColumnHeaders(t, result) {
-    for(var i = 0; i < result.servers.length; i++)
+    for(var i = 0; i <= result.servers.length; i++)
```

Comment:
Why? This is now iterating `result.servers.length + 1` times.

Reply:
Because the first `<colgroup>` groups the `<th>`es in the first column. Thus one `<colgroup>` was missing for the last `<td>`-column.